crysthack_day1_GitHub_workflow_exercise/names/matthewrowles.txt

```diff
@@ -1 +1,2 @@
 Matthew Rowles
+Perth, Western Australia
```

Comment:
```suggestion
Perth, Western Australia, Planet Earth
```
People can also give you feedback in form of a suggestion about how to modify the code. However, if you would accept this change and commit it, you would then have to make sure that you update your local copy. That might be a little bit more advanced, so do not worry about it just wanted to show it to you. Hope @sbillinge thinks it too :)

Reply:
yes, this is right Claudia.  It is sometimes less work to do the edit than to explain to the person how to do the edit, but generally I prefer to do the work to explain the edit.  It leads to more discussion and better code, and better coders (often the reviewer and the reviewee learn something!).   It can also lead to more conflicts doing it the other way.  I designed the exercise to avoid conflicts (multiple people suggesting edits to the same line of code) because that can get head spinning very quickly.

All these things are much less head-spinning if you are really remembering and understanding where the code is physically and where it is logically, e.g., "in the git database on my hardrive, in a git database in my user account on github in a pole-barn in iowa" and so on, which is why I spent a bit of time on that.    Also, remembering which is the canonical database, and the goal is on the one hand to get changes accepted into the canonical datbase and on the other hand, remembering to synch local databases with the canonical....

Reply:
great discussion.... @rowlesmr this is really head-spinning, and there is a steep learning curve....but as i said in my talk, it is the gift that keeps on giving, it is really worth the effort.